tst/testspecial/low-mem-plist-1.g.out

```diff
@@ -0,0 +1,17 @@
+gap> l := [];
+[  ]
+gap> l[2^50] := 1;
+Error, Cannot allocate 9007199254741000 bytes: cannot extend the workspace any more!!!!
```

Comment:
Won't this test break in 32bit mode, because GAP only allows indices which are immediate integers? At least I get this in my 64bit GAP:
```
gap> l:=[];;l[2^100]:=1;
Error, List Assignment: <pos> must be a positive small integer (not a large positive integer)
```

Reply:
Yes, testing in 32-bit is going to be a pain, and probably require splitting testspecial so we can have special 32-bit tests.